src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -255,6 +259,9 @@ public String delete(String id, AuthenticatedUserRequest userReq) {
         .newJob(description, DatasetDeleteFlight.class, null, userReq)
         .addParameter(JobMapKeys.DATASET_ID.getKeyName(), id)
         .addParameter(JobMapKeys.CLOUD_PLATFORM.getKeyName(), platform.name())
+        .addParameter(JobMapKeys.IAM_RESOURCE_TYPE.getKeyName(), IamResourceType.DATASET)
```

Comment:
I wonder (totally fine in a followup if we think it's useful): should we have method that add the three of these? I just wonder if it might make it less likely to put in an incomplete ID.